brave/browser-laptop
# ledger recovery should be aware of network status and localization

Fix #4720
Fix #5806

## Test Plan

  1. Go to Preferences > Payments
  2. Click Advanced Settings
  3. Click Recover Your Wallet
  4. Try to recover with the wrong keys
  5. You should see the "Recovery Failed" overlay
  6. Repeat steps 1-3
  7. Turn off network connection
  8. Try to recover with right or wrong keys
  9. You should see the "Network Error" overlay

## Auditors

@mrose17 can you check this logic? typeof object = network problems else data input problem?
@bradleyrichter how does the copy look for (Network Error - Please check your internet connection and try again.)
@bsclifton can you give me a review for the JS code? :) Is there a way to test the network status?

<img width="718" alt="screen shot 2016-12-29 at 9 19 22 pm" src="https://cloud.githubusercontent.com/assets/490294/21560043/e24334ec-ce0c-11e6-9ffa-faa118957af0.png">
<img width="735" alt="screen shot 2016-12-29 at 9 21 58 pm" src="https://cloud.githubusercontent.com/assets/490294/21560044/e243e89c-ce0c-11e6-9131-218aef58dfbe.png">
